rustls/src/client/mod.rs

```diff
@@ -620,6 +617,55 @@ impl ClientSessionImpl {
     }
 }
 
+// Added in anticipation of another struct, EncryptedHost, that returns
+// Some() for the last two methods.
+pub trait HelloData {
+    fn get_extra_exts(&self) -> &Vec<ClientExtension>;
```

Comment:
If we're going to be sticking to the API guidelines more closely (#541), let's not add [new `get_` prefixed getters](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter).

Reply:
Got it. Also not in the patch anymore.